### DISCUSSION TITLE
RGB 0.11 beta4 workflow

### DISCUSSION THREAD

**Discussion by deepseeksss** [RGB-WG/rgb#94] [Q&A] [answered]
1. bob create an invoice, such as `rgb:xo5Z5s4-aM15dX9rn-reGdVKaFW-kkrYj2ruZ-rqDGVjVHF-SzpzS3/RGB20/1000+tb:pz7994vg9lktzmhs437f65x96t09k5vecju8gmj8jguswav5el6wqnhfz5z`

The `tb:pz7994vg9lktzmhs437f65x96t09k5vecju8gmj8jguswav5el6wqnhfz5z` above is bob's address, is right?

2. alice accept the invoice 
 After alice accept the invoice, there is a new psbt and a new consignment.
 
3. bob accept the consignment from step 2
 
 After finished these steps, I use sparrow to sign and broadcast the psbt and waiting the new transaction confirmed.

4.  bob state the contract, the output like blow:
```
RGB: command-line wallet for RGB smart contracts
     by LNP/BP Standards Association

Loading descriptor from wallet default ... success
Loading stock ... success
Global:
  spec := (naming=(ticker=("DBG"), name=("Debug asset"), details=1(("Pay attention: the asset has no value"))), precision=2)
  data := (terms=("SUBJECT TO, AND WITHOUT IN ANY WAY LIMITING, THE REPRESENTATIONS AND WARRANTIES OF ANY SELLER  EXPRESSLY SET FORTH IN THIS AGREEMENT OR ANY OTHER EXPRESS OBLIGATION OF SELLERS PURSUANT TO THE TERMS HEREOF, AND ACKNOWLEDGING THE PRIOR USE OF THE PROPERTY AND PURCHASER’S OPPORTUNITY  TO INSPECT THE PROPERTY, PURCHASER AGREES TO PURCHASE THE PROPERTY “AS IS”, “WHERE IS”,  WITH ALL FAULTS AND CONDITIONS THEREON. ANY WRITTEN OR ORAL INFORMATION, REPORTS, STATEMENTS,  DOCUMENTS OR RECORDS CONCERNING THE PROPERTY PROVIDED OR MADE AVAILABLE TO PURCHASER, ITS AGENTS OR CONSTITUENTS BY ANY SELLER, ANY SELLER’S AGENTS, EMPLOYEES OR THIRD PARTIES REPRESENTING OR PURPORTING TO REPRESENT ANY SELLER, SHALL NOT BE REPRESENTATIONS OR WARRANTIES, UNLESS SPECIFICALLY SET FORTH HEREIN. IN PURCHASING THE PROPERTY OR TAKING OTHER ACTION HEREUNDER, PURCHASER HAS NOT AND SHALL NOT RELY ON ANY SUCH DISCLOSURES, BUT RATHER, PURCHASER SHALL RELY ONLY ON PURCHASER’S OWN INSPECTION OF THE PROPERTY AND THE REPRESENTATIONS AND WARRANTIES  HEREIN. PURCHASER ACKNOWLEDGES THAT THE PURCHASE PRICE REFLECTS AND TAKES INTO ACCOUNT THAT THE PROPERTY IS BEING SOLD “AS IS”.
"), media=~)
  issuedSupply := (100000000)
  created := (1687969158)

Owned:
  assetOwner:
```
the owner is empty, it should be alice, and there must be a new owner which is bob with transfer value (1000)

another question:
The transaction in psbt, there are two outputs, first one is alice, and another one is bob's address （which I mentioned in step 1）, 
the result is alice's btc is transfer to alice and bob, is it right ?

**Comment by cymqqqq** [comment, marked as answer]
1. The tb:pz7994vg9lktzmhs437f65x96t09k5vecju8gmj8jguswav5el6wqnhfz5z above is bob's address, is right?

So tb is the blinded utxo of the output of bob's uxto, yes, it's a blinded address. Bob is the person who wants to claim this asset(First, he must generate an invoice by his utxo).
2.There is a new psbt. I think it's not a new PSBT, it's the same PSBT but needs both the issuer and the receiver to sign.
3.The owner is empty, maybe there is no utxo in your off-chain RGB database.
